src/fileProviders/FSProvider.ts

```diff
@@ -0,0 +1,50 @@
+import { copyFile, mkdir, rm } from 'fs/promises';
+import { GraaspFileItemOptions } from 'graasp-plugin-file-item';
+import { Sharp } from 'sharp';
+import FileOperations from './FileOperations';
+import { createFsFolder, createFsKey } from '../utils/helpers';
+
+export class FSProvider implements FileOperations {
+  private readonly options: GraaspFileItemOptions;
+  private readonly prefix;
+
+  constructor(options: GraaspFileItemOptions, prefix: string) {
+    this.options = options;
+    this.prefix = prefix;
+  }
+
+  async copyObject(
+    originalId: string,
+    newId: string,
+    _memberId: string,
+    size: string,
+  ): Promise<void> {
```

Comment:
What about using an object?

Reply:
It seems it doesn't follow the interface order, does it?

Reply:
I don't think using an object will give any benefits